### ISSUE TITLE
Day 1 Deliverables

### ISSUE THREAD

**Issue by srslie** [srslie/PD-Pal#3]
# Day 1 Deliverables

Create a DM with both instructors, send the following:
- [x] Your niche audience (be as specific and narrow as possible)
  - Students and recent grads of Turing’s Front-End Development track who are looking for jobs, prospective students
- [x] Your chosen API
  - https://jobs.github.com/api or https://opensource.indeedeng.io/api-documentation/docs/job-search/
- [x] The information you will be using from it
   - job listings for with query parameters of the skillset of front-end grads (javascript, react, react router, OOP, git, github, RESTful APIs, html5, css, scss)
- [x] A list of your app’s MVP features
  - Display of job listings
  - ability for user to favorite 
  - mark jobs as ‘applied to’
- [x] Optional: a list of your app’s nice-to-have features
  - Additional search parameters like location
  - User authorization/save their data
  - Integration of PD prompts/advice
- [x] A backup API & audience
  - https://cloud.google.com/natural-language/docs/
  - Job seekers who are bad at cover letters, allows analysis of letters and storing of approved snippets

 Once your project MVP is approved (instructors will help you narrow down your MVP to a reasonable scope), please also send the following:
- [x] Wireframes
  - [miro](https://miro.com/welcomeonboard/0mpcPUX9FJebnAsmONXP9TpktAuBVp70lJ2NPOlLDLQBZyO8Q75rgF6ZcV9Yugmu)
- [x] Links to 2 design inspirations
   - [huntr](https://huntr.co/)
  - [linkedin jobs]( https://www.linkedin.com/jobs/)
- [x] 2 separate user personas describing people in your niche audience
  - [persona google doc](https://docs.google.com/document/d/1sMR0UBakjWWw8B11Cf-bvOpeBQuuLOEJrBUaM99cgV0/edit?usp=sharing)
- [x] Project board link
  - [github project](https://github.com/srslie/PDPal/projects)
- [x] Repo link
  - [github repo](https://github.com/srslie/PDPal)
- [x]  How does this app illustrate your personal interests and/or your career vision? How does it fit into the story you want to tell potential employers about yourself?
  -  This fits into my career vision both because it is centered around getting jobs, as well as being very tangible and helpful by serving people, which is important to me. It also shows I’m creative at solving a combination of needs (in this case class assignments and PD/job-hunting).
